lib/urlwatch/jobs.py

```diff
@@ -187,6 +188,25 @@ def get_location(self):
         return self.url
 
     def retrieve(self, job_state):
+        if hasattr(self, 'tries') is False:
+            self.tries = 3
+        if hasattr(self, 'retry_pause') is False:
+            self.retry_pause = 20
```

Comment:
Not sure if we should by default have a 20 second (* 3 retries) pause. This could really prolong the runtime of the script. Can you make it opt-in instead (only 1 retry by default, or having a config-global default value and being able to override per job)? Also, usually back-off is implemented using exponential timeouts (e.g. 2, 4, 8, 16 seconds) -- see https://en.wikipedia.org/wiki/Exponential_backoff